README.md

```diff
@@ -113,7 +113,13 @@ kube-state-metrics exposes its own metrics under `--telemetry-host` and `--telem
 
 ### Resource recommendation
 
-Resource usage changes with the size of the cluster. As a general rule, you should allocate
+Resource usage for kube-state-metrics changes with the size, i.e., the node number, of the cluster.
```

Comment:
I think it's actually a problem that it doesn't exactly scale with the number of nodes, but rather with the total number of objects, which can vastly vary even at the same node count. I think we should mention that here.